### PR TITLE
Fix exploits of undefined behaviors in our own library

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
CI only for now.

**Comment by sjrd** [comment]
Review by @gzm0

File: library/src/main/scala/scala/scalajs/runtime/StackTrace.scala [hunk 1]
```diff
@@ -130,8 +132,7 @@ object StackTrace {
           jsSte.fileName, jsSte.lineNumber)
 
       jsSte.columnNumber foreach { cn =>
-        // Store column in magic field
-        ste.asInstanceOf[js.Dynamic].columnNumber = cn
+        ste.setColumnNumber(cn)
```
Comment (gzm0) [inline library/src/main/scala/scala/scalajs/runtime/StackTrace.scala]: `jsSte.columnNumber foreach(ste.setColumnNumber)`?

Comment (sjrd) [inline library/src/main/scala/scala/scalajs/runtime/StackTrace.scala]: OK.

**Comment by gzm0** [comment]
That's all.

**Comment by sjrd** [comment]
Updated.

**Comment by gzm0** [comment]
LGTM

**Pull request merged**